boring/src/ssl/mod.rs

```diff
@@ -1177,6 +1185,23 @@ impl SslContextBuilder {
         }
     }
 
+    /// Sets the list preferred curves for the context to be the colon-separated list |curves|.
+    /// Each element of `curves` should be a curve name (e.g. P-256, X25519, ...).
```

Comment:
Nit: I'd just give a concrete example; eg. `P-256:X25519`.

Reply:
Good idea, done. FWIW this paragraph was copied from boringSSL's API docs :D